index.js

```diff
@@ -199,6 +200,10 @@ class ServerlessPlugin {
       args.unshift(TAG);
     }
 
+    // Avoid logging the same resolved layer multiple times
```

Comment:
Please don't place business logic for specific use-case here. Think `log` method as `console.log`. This method should be for general logging. Move this to `replaceLayerVersions` method implementation instead.

Reply:
1. declare set which stores "resolved" layer arn
2. if set does not contain given layer arn, log & store given layer arn, otherwise, just skip.

and It would be appreciated if you would implement "log level" option.

Reply:
Hey, I've updated the PR. 
I just didn't do anything about "log level" because I didn't quite get what this option would do.

Reply:
I thought we can implement a plugin option for controlling log output verbosity.

For example, 
- INFO: Most verbose logging output. Prints any informational log (e.g. prints entire layer replacements without deduplication, with entire context including function name and layer arn)
- WARN: Prints warnings only. Prints warnings only (e.g. prints deduplicated layer replacements - which current PR does)
- ERROR: Prints errors only. Prints errors only (e.g. unknown layer arn)

Then, Users can control log verbosity like this:

```yaml
// serverless.yml

// ... (truncated)
plugins:
  - serverless-latest-layer-version
custom:
  latestLayerVersion:
    logLevel: WARN
```

Reply:
Thank you for creating an issue for that 👍